pixart.js

```diff
@@ -1 +1,13 @@
+$(document).on("ready", function()
+{
+var setcolor=$("#set-color");
+setcolor.on("click", changeBoxToInput);
 
+var input=$(":input").val();
+
+
+function changeBoxToInput(){
+  $(".brush").css("background", ":input");
```

Comment:
I can see that you had the concept down, but it's just the syntax is off, and you might be trying to get too much out of that one line of code.

We want to change the css of the `.square`  `<div>`s, but we want to grab the `background` property from the `.brush` `<div>`